src/files/file-icon/FileIcon.js

```diff
@@ -11,23 +11,23 @@ import DocText from '../../icons/GlyphDocText'
 
 const style = { width: 36 }
 
-export default function FileIcon ({ name, type }) {
+export default function FileIcon ({ name, type, cls }) {
```

Comment:
give `cls` a default value, ot `undefined` appears in the class list.

```suggestion
export default function FileIcon ({ name, type, cls = '' }) {
```